docs/drash/v2.x/2_tutorials/6_services/4_drash_approved_services/graphql.md

```diff
@@ -47,110 +52,146 @@ version can be found [here](https://github.com/drashland/drash/releases/latest).
 
-const root = {
-  greeting: () => { // or greeting: (source, args, context, info) => {
-    return "Hello world!";
-  },
-};
+   const schema = GraphQL.buildSchema(`
+     type Query {
+       greeting: String
+     }
+   `);
 
-// Instantiate the GraphQL service
+   const root = {
+     greeting: () => { // or greeting: (source, args, context, info) => {
+       return "Hello world!";
+     },
+   };
 
-const graphQl = new GraphQLService({
-  schema,
-  graphiql: true,
-  rootValue: root,
-});
+   // Instantiate the GraphQL service
 
-// Create your GraphQL resource so that:
-//
-//   1. The playground can be viewed and used.
-//   2. Clients can make queries to your graph.
-//
-// Also, note that `services` includes the instantiated GraphQLService.
+   const graphQl = new GraphQLService({
+     schema,
+     graphiql: true,
+     rootValue: root,
+   });
 
-class GraphQLResource extends Drash.Resource {
-  public paths = ["/graphql"];
+   const server = new Drash.Server({
+     hostname: "localhost",
+     port: 1447,
+     protocol: "http",
+     resources: [],
+     services: [graphQl],
+   });
 
-  public services = {
-    ALL: [graphQl],
-  };
+   server.run();
 
-  public GET(request: Drash.Request, response: Drash.Response): void {
-    // This is intentionally left blank.
-    //
-    // This is only defined to allow GET requests to the front-end playground.
-    // Without this, Drash will throw a 405 Method Not Allowed error when
-    // requesting to view the playground at /graphql.
-    //
-    // If you want to secure your playground, then you can fill in this method
-    // with logic to secure it. For example:
-    //
-    // if (request.headers.get("some-token")! !== "someSecret") {
-    //   response.status = 401;
-    //   return response.text("You do not have access to this resource.");
-    // }
-  }
-
-  public POST(request: Drash.Request, response: Drash.Response): void {
-    // This is intentionally left blank.
-    //
-    // This is only defined so that POST requests to this resource can be
-    // processed. Without this, Drash will throw a 405 Method Not Allowed error
-    // when clients try to make GraphQL queries.
-  }
-}
-
-// Create and run your server
-
-const server = new Drash.Server({
-  hostname: "localhost",
-  port: 1447,
-  protocol: "http",
-  resources: [GraphQLResource],
-});
-
-server.run();
-
-console.log(`Server running at ${server.address}.`);
-```
+   console.log(`Server running at ${server.address}.`);
+   ```
 
 ## Verification
 
 1. Run your app.
 
-```shell
-$ deno run --allow-net app.ts
-```
+   ```shell
+   $ deno run --allow-net app.ts
+   ```
 
-2. View your front-end playground by going to `http://localhost:1447/graphql`.
+1. View your front-end playground by going to `http://localhost:1447/graphql`.
 
-3. Make a query by entering the following in the left pane of your playground.
+1. Make a query by entering the following in the left pane of your playground.
```

Comment:
two `1.`'s, should be `1.`, `2.`?

Reply:
dude i found out last week that markdown automatically changes the numbers so you only need to do 1. 1. 1. 🤯

Reply:
:o